osu.ElasticIndexer/SoloScoreIndexer.cs

```diff
@@ -28,16 +28,21 @@ public void Run(CancellationToken token)
 
                 checkSchema();
 
+                redis.AddActiveSchema(AppSettings.Schema);
+
                 using (new Timer(_ => checkSchema(), null, TimeSpan.Zero, TimeSpan.FromSeconds(5)))
                     new IndexQueueProcessor(metadata.Name, client, Stop).Run(cts.Token);
             }
         }
 
-        public void Stop()
+        public void Stop(bool aborted)
```

Comment:
Rather than adding this parameter, I think it would be better to just do an explicit call in `checkSchema()` as that's the only case where the active schema should be removed.